sentry-ruby/lib/sentry/configuration.rb

```diff
@@ -509,7 +559,6 @@ def profiler_class=(profiler_class)
         begin
           require "vernier"
         rescue LoadError
-          raise ArgumentError, "Please add the 'vernier' gem to your Gemfile to use the Vernier profiler with Sentry."
```

Comment:
This is behavioral change - not sure if we want to keep it as it was? If we keep raising it will be inconsistent with Stackprof profiler, where it just log_warn.

Reply:
fine with not raising